include/swift/AST/NameLookup.h

```diff
@@ -225,6 +225,8 @@ enum class UnqualifiedLookupFlags {
   /// This lookup should include results from outside the innermost scope with
   /// results.
   IncludeOuterResults   = 1 << 4,
+  /// Includes property wrapper name lookup results
+  IncludePropertyWrapperResults = 1 << 5,
```

Comment:
It's not entirely clear to me when you want this turned off. Do you mind adding a comment or just explaining here if it's totally obvious?

Reply:
I had added this flag before you removed the other implementation of unqualified lookup, so the property wrapper lookup was just performing unqualified lookup again with the prefixed names (which seemed expensive so I was limiting it to resolving decl references in the type checker). I think this can just be removed now